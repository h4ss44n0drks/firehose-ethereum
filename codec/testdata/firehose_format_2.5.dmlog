FIRE INIT 2.5 geth 1.10.17-dm-stable
FIRE BEGIN_APPLY_TRX 74b9554595faea99f11f5310c086acaec4686ef94b30466f966d67071b228b65 79f19a189068a79ec01343b8c70096114f048753 . 29 9b7b4ad21ac016e1766749e203d4776d351ac9c7d56cdd9b3c2d5b4f940c1f7e 5b08a942dbc15ff47f7f40d4519fe2e20948579b0baebaca66a7b651f6f483ed 966000 0ba43b7400 110 f93d3def0000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000004b7265616c79206c6f6e6720737472696e67206c6172676572207468616e20333220627974657320746f2074657374206f757420736f6c69646974792073706c697474696e67207374756666000000000000000000000000000000000000000000 0312345678901234567890123456789012345678900200000000000000000000000000000000000000000000000000000000000000ab00000000000000000000000000000000000000000000000000000000000000efabcdefabcdefabcdefabcdefabcdefabcdef0910001234567890123456789012345678901234567890010000000000000000000000000000000000000000000000000000000000000012 fe 56 3 1 0 99 ab 0174b9554595faea99f11f5310c086acaec4686ef94b30466f966d67071b228b65,0184b9554595faea99f11f5310c086acaec4686ef94b30466f966d67071b228b64
FIRE TRX_FROM ab07a50ad459b41fe065f7bbab866d5390e9f705
FIRE END_APPLY_TRX 23358 . 23358 00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 11 0 0 []
