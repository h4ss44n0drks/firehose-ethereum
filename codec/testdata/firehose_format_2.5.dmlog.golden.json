[
  {
    "accessList": [
      {
        "address": "1234567890123456789012345678901234567890",
        "storageKeys": [
          "00000000000000000000000000000000000000000000000000000000000000ab",
          "00000000000000000000000000000000000000000000000000000000000000ef"
        ]
      },
      {
        "address": "abcdefabcdefabcdefabcdefabcdefabcdef0910"
      },
      {
        "address": "1234567890123456789012345678901234567890",
        "storageKeys": [
          "0000000000000000000000000000000000000000000000000000000000000012"
        ]
      }
    ],
    "beginOrdinal": "1",
    "blobGas": "99",
    "blobGasFeeCap": "ab",
    "blobHashes": [
      "0174b9554595faea99f11f5310c086acaec4686ef94b30466f966d67071b228b65",
      "0184b9554595faea99f11f5310c086acaec4686ef94b30466f966d67071b228b64"
    ],
    "calls": [
      {
        "address": "79f19a189068a79ec01343b8c70096114f048753",
        "callType": "CALL",
        "caller": "ab07a50ad459b41fe065f7bbab866d5390e9f705",
        "index": 1
      }
    ],
    "endOrdinal": "11",
    "from": "ab07a50ad459b41fe065f7bbab866d5390e9f705",
    "gasLimit": "966000",
    "gasPrice": "0ba43b7400",
    "gasUsed": "23358",
    "hash": "74b9554595faea99f11f5310c086acaec4686ef94b30466f966d67071b228b65",
    "input": "f93d3def0000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000004b7265616c79206c6f6e6720737472696e67206c6172676572207468616e20333220627974657320746f2074657374206f757420736f6c69646974792073706c697474696e67207374756666000000000000000000000000000000000000000000",
    "maxFeePerGas": "fe",
    "maxPriorityFeePerGas": "56",
    "nonce": "110",
    "r": "9b7b4ad21ac016e1766749e203d4776d351ac9c7d56cdd9b3c2d5b4f940c1f7e",
    "receipt": {
      "blobGasPrice": "00",
      "blobGasUsed": "0",
      "cumulativeGasUsed": "23358",
      "logsBloom": "00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
    },
    "s": "5b08a942dbc15ff47f7f40d4519fe2e20948579b0baebaca66a7b651f6f483ed",
    "to": "79f19a189068a79ec01343b8c70096114f048753",
    "type": "TRX_TYPE_BLOB",
    "v": "29"
  }
]